.gitattributes

```diff
@@ -1 +1,18 @@
-/.pnp.js binary
+# This line does two things:
+# - enable git's CRLF -> LF modification
+# - tell git to guess whether files are text or binary
+*                       text=auto
```

Comment:
Not sure if this is safe for the windows builds to be honest, because this will cause git to checkout files with CRLF as line ending on that platform. We could add `eol=lf` to always force git to use LF line endings, or we could just skip this directive altogether.

I copied this over from literally every gitattributes file I've ever written at work, where I got tired of people committing CRLFs into our repos even though we all use macOS.

Reply:
Windows builds are successful so I'd assume it's safe&hellip;